mycroft/metrics/__init__.py

```diff
@@ -104,9 +109,10 @@ def publish():
 
 
 class MetricsPublisher(object):
-    def __init__(self, url=config.get("url"), enabled=config.get("metrics")):
-        self.url = url
-        self.enabled = enabled
+    def __init__(self, url=None, enabled=None):
```

Comment:
Nitpicking:
I think `False` should be used instead of `None` the effect is the same but slightly more readable. (should be immutable just as None so no issue to have as default for the argument.